sdk/clientcore/core/src/main/java/io/clientcore/core/http/pipeline/HttpInstrumentationPolicy.java

```diff
@@ -179,8 +186,11 @@ public final class HttpInstrumentationPolicy implements HttpPipelinePolicy {
      * @param instrumentationOptions Application telemetry options.
      */
     public HttpInstrumentationPolicy(HttpInstrumentationOptions instrumentationOptions) {
-        Instrumentation instrumentation = Instrumentation.create(instrumentationOptions, LIBRARY_OPTIONS);
-        this.tracer = instrumentation.getTracer();
+        this.instrumentation = Instrumentation.create(instrumentationOptions, LIBRARY_OPTIONS);
+        this.tracer = instrumentation.createTracer();
+        this.meter = instrumentation.createMeter();
+        this.httpRequestDuration
+            = meter.createDoubleHistogram("http.client.request.duration", "Duration of HTTP client requests", "s");
```

Comment:
Mind adding a comment to this line so it's clearer. Right now, without being an expert here this looks to me that the metric is `http.client.request.duration` with a description of `Duration of HTTP client requests`, but `s` eludes me thinking it means the metric is in seconds.